pkg/utils/utils.go

```diff
@@ -78,3 +80,28 @@ func DefaultIfZeroFloat(left, right float32) float32 {
 func DefaultIfZeroFloat64(left, right float64) float64 {
 	return defaultIfZero(left, right).(float64)
 }
+
+func getRootBlockFile(devName string) string {
+	switch {
+	case strings.HasPrefix(devName, sys.HostDevRoot):
+		return devName
+	case strings.Contains(devName, sys.DirectCSIDevRoot):
+		return getRootBlockFile(filepath.Base(devName))
+	default:
+		name := strings.ReplaceAll(
+			strings.Replace(devName, sys.DirectCSIPartitionInfix, "", 1),
+			sys.DirectCSIPartitionInfix,
+			sys.HostPartitionInfix,
+		)
+		return filepath.Join(sys.HostDevRoot, name)
+	}
+}
+
+func GetDrivePath(drive *directcsi.DirectCSIDrive) string {
```

Comment:
We don't need to switch case here on versions, because, there can be v1beta2 drives as well with the `Path` set to `/var/lib/direct-csi`  notation

Any fresh installations from v1.4.0 also will have `/var/lib/direct-csi` notation on `Path`

We can directly use getRootBlockFile wherever needed

Reply:
Done